js/media/fieldmanager-media.js

```diff
@@ -78,15 +78,21 @@ $( document ).on( 'click', '.fm-media-button', function( event ) {
 		props.linkUrl = '#';
 		props.caption = '';
 		$el.parent().find('.fm-media-id').val( attachment.id );
+		var preview = '<div class="media-file-preview">';
 		if ( attachment.type == 'image' ) {
 			props.url = props.src;
-			var preview = 'Uploaded file:<br />';
 			preview += wp.media.string.image( props );
+		} else if ( attachment.type == 'audio' ) {
+			preview += '<a href="#"><span class="dashicons dashicons-media-audio"></span></a>';
+		} else if ( attachment.type == 'video' ) {
+			preview += '<a href="#"><span class="dashicons dashicons-media-video"></span></a>';
 		} else {
-			var preview = 'Uploaded file:&nbsp;';
-			preview += wp.media.string.link( props );
+			preview += '<a href="#"><span class="dashicons dashicons-media-document"></span></a>';
```

Comment:
Is there a risk of these media icon classes becoming out of sync with the filtered icon class used in PHP?

Reply:
@dlh01 it is possible. I pushed up a change that will set the icon depending on the attachment icon type. See https://github.com/alleyinteractive/wordpress-fieldmanager/pull/468/files#diff-7f0fb55a63140bb4d44360770c1ec06ff18be39f6c9e359f8f7370bca480efcbR81-R83

Let me know what you think about his change.

Reply:
We are now relying on the media type from the attachment to output the correct icon.